redis/asyncio/connection.py

```diff
@@ -953,6 +978,20 @@ def to_bool(value) -> Optional[bool]:
     return bool(value)
 
 
+def parse_ssl_verify_flags(value):
+    # flags are passed in as a string representation of a list,
+    # e.g. VERIFY_X509_STRICT, VERIFY_X509_PARTIAL_CHAIN
+    verify_flags_str = value.replace("[", "").replace("]", "")
```

Comment:
I think even this is not needed. According to a URL query parameters spec it is possible to pass array like this:

`?foo[]=bar&foo[]=baz` and I expect that url parsing function will return you a list `foo => [bar, baz]`. Could you please check it?